syntax = "proto3";

package math;

service MathFormula {
    rpc GetFormula(RequestMessage) returns (ReplyMessage) {}
}

message RequestMessage {
    string message = 1;
}

message ReplyMessage {
    string message = 1;
}